### DISCUSSION TITLE
11/28 다음 주 글 초안 공유하기

### DISCUSSION THREAD

**Discussion by Orm-camp** [Orm-camp/blog-feedback#42] [동글동글 3기 피드백]
다음 주 글 주제나 정리 중인 초안을 공유해 주세요! (주제와 소제목 정도 수준으로)
다른 사람들은 어떤 글을 어떻게 구성하고 있는지 보면서 시야를 넓혀나가 볼까요? 👀

**Comment by mh0223** [comment]
주제: 자바스크립트 실행 컨텍스트
제목: [스터디] 자바스크립트 실행 컨텍스트란? 

스터디에서 공유할 내용을 블로그에 적어보려구 합니다,,

**Comment by ghost** [comment]
주제 : MongoDB로 만든 간단한 프로젝트 후기

**Comment by mycreature** [comment]
주제: 백엔드 수업내용중 DB, SQL 정리

**Comment by hansol-woff** [comment]
초안을 써본 것은 아니지만 Three.js의 기초적인 부분 부터 차근차근 정리하는걸 주제로 해볼까 합니다..
